meinberlin/assets/scss/components/_timeline.scss

```diff
@@ -78,11 +78,7 @@ $circle-size: 0.8em;
         &:last-child:before {
             width: auto;
             height: 1px;
-
-            left: -15px;
-            right: -15px;
-            top: auto;
-            bottom: -$circle-size;
+            inset: auto -15px - $circle-size -15px;
```

Comment:
it forced me to do a whitespace between the minus and `$circle-size`, not sure if that's actually correct